credly/views.py

```diff
@@ -0,0 +1,127 @@
+from django.contrib.auth.models import User
+from django.shortcuts import render
+
+# Create your views here.
+
+import base64
+from django.views.decorators.csrf import csrf_exempt
+from requests.auth import AuthBase
+import slumber
+from django.http import JsonResponse
+from django.views.generic import View
+from slumber.exceptions import HttpNotFoundError
+
+
+class ApiAuth(AuthBase):
+    def __init__(self, username, api_key):
+        self.username = username
+        self.api_key = api_key
+
+    def __call__(self, r):
+        base64string = base64.encodestring('%s:%s' % ("issa.mh.khalil@gmail.com", "5668c9ded6745"))[:-1]
+        auth_header = "Basic %s" % base64string
+
+        r.headers['X-Api-Key'] = self.username
+        r.headers['X-Api-Secret'] = self.api_key
+        r.headers['Authorization'] = auth_header
+
+        return r
+
+
+class CredlyView(View):
+    credly = None
+    route_base = None
+
+    def __init__(self):
+        key = "64166cde88e2816feaef7f7ff74a5687"
+        secret = "iZGR1FZ6KAKxhj3QWUu1iNmXhjs0wZbEbnYTKng5Np0r9t/mtyofqfIAjTKfVmV2cdTw1o7nH6rp4u1CGq8hPEiWhZecEOTGyAX7XAaPPnTh+1MVGADHYJgeqdR0snyOAJbkODA2LOhW9n+2eLMnlmGDA3fUbCdk6oZbPnYJqP4="
```

Comment:
to add to this: github is constantly getting mined for credentials. These should be considered leaked and get invalidated.